docs/changes/latest.inc

```diff
@@ -10,7 +10,7 @@
 
 .. NOTE: add the contributors and reference to the github issue/PR at the end
    Example:
-      - Implemented feature X (:gh:`151` by `Sami Hamdan`_).
+   - Implemented feature X (:gh:`151` by `Sami Hamdan`_).
```

Comment:
Is this still part of the note? Or it will become a new element in the list?

Reply:
I wasn't sure about this either and just fixed the indentation.

Reply:
I thought fede added this as an example?

Reply:
Reverting the change.